conf/docker/build.sh

```diff
@@ -51,7 +51,10 @@ echo Images are being built for registry org/username \"$HUBORG\" with the tag \
 # Build dataverse-solr
 #
 # Use "conf" directory as context so we can copy schema.xml into Solr image.
```

Comment:
Am I right in thinking we don't need this comment about the "conf" directory anymore? If so, please remove it.